src/EventBuilder/Program.cs

```
@@ -21,133 +24,128 @@ internal static class Program
-                        break;
+                        await ExtractEventsFromAssemblies(options.OutputPath, platform).ConfigureAwait(false);
 
-                    case AutoPlatform.XamForms:
-                        platform = new XamForms();
-                        break;
-
-                    case AutoPlatform.Tizen:
-                        platform = new Tizen();
-                        break;
-
-                    case AutoPlatform.UWP:
-                        platform = new UWP();
-                        break;
-
-                    case AutoPlatform.Winforms:
-                        platform = new Winforms();
-                        break;
-
-                    case AutoPlatform.Essentials:
-                        platform = new Essentials();
-                        _mustacheTemplate = "XamarinEssentialsTemplate.mustache";
-                        break;
-
-                    default:
-                        throw new ArgumentException($"Platform not {options.Platform} supported");
+                        Environment.Exit((int)ExitCode.Success);
                     }
-
-                    ExtractEventsFromAssemblies(platform);
-
-                    Environment.Exit((int)ExitCode.Success);
-                }
-                catch (Exception ex)
-                {
-                    Log.Fatal(ex.ToString());
-
-                    if (Debugger.IsAttached)
+                    catch (Exception ex)
                     {
-                        Debugger.Break();
+                        Log.Fatal(ex.ToString());
+
+                        if (Debugger.IsAttached)
+                        {
+                            Debugger.Break();
+                        }
                     }
-                }
-            }
 
-            Environment.Exit((int)ExitCode.Error);
+                    Environment.Exit((int)ExitCode.Error);
+                },
+                _ => Task.CompletedTask).ConfigureAwait(false);
```

Really? Ignoring the errors? :(